admin-ui/plugins/auth-server/components/AuthN/DefaultAcr.js

```diff
@@ -0,0 +1,134 @@
+import React, { useContext, useEffect, useState, useMemo } from 'react'
+import { useSelector, useDispatch } from 'react-redux'
+import { useCedarling } from '@/cedarling'
+import GluuViewWrapper from 'Routes/Apps/Gluu/GluuViewWrapper'
+import { useTranslation } from 'react-i18next'
+import { ACR_READ, ACR_WRITE } from 'Utils/PermChecker'
+import SetTitle from 'Utils/SetTitle'
+import { getAcrsConfig, editAcrs } from 'Plugins/auth-server/redux/features/acrSlice'
+import { getAgama } from 'Plugins/auth-server/redux/features/agamaSlice'
+import GluuCommitDialog from 'Routes/Apps/Gluu/GluuCommitDialog'
+import { buildPayload } from 'Utils/PermChecker'
+import { Button, Form } from 'Components'
+import GluuLoader from '@/routes/Apps/Gluu/GluuLoader'
+import { ThemeContext } from '@/context/theme/themeContext'
+import DefaultAcrInput from '../Configuration/DefaultAcrInput'
+import { getScripts } from 'Redux/features/initSlice'
+import { SIMPLE_PASSWORD_AUTH } from 'Plugins/auth-server/common/Constants'
+
+function DefaultAcr() {
+  const { hasCedarPermission, authorize } = useCedarling()
+  const dispatch = useDispatch()
+
+  const { acrReponse: acrs, loading: acrLoading } = useSelector((state) => state.acrReducer)
+  const { scripts, loading: scriptLoading } = useSelector((state) => state.initReducer)
+  const { agamaList, loading: agamaLoading } = useSelector((state) => state.agamaReducer)
+
+  const { t } = useTranslation()
+
+  const [modal, setModal] = useState(false)
+  const [put, setPut] = useState(null)
+  const userAction = {}
+  const theme = useContext(ThemeContext)
+  const selectedTheme = theme.state.theme
+
+  SetTitle('ACR Management')
+
+  useEffect(() => {
+    const initializeAcr = async () => {
+      try {
+        await authorize([ACR_WRITE, ACR_READ])
+      } catch (error) {
+        console.error('Error authorizing ACR permissions:', error)
+      }
+    }
+
+    initializeAcr()
+    dispatch(getScripts({ action: userAction }))
+    dispatch(getAgama())
+    dispatch(getAcrsConfig())
+  }, [authorize, dispatch])
+
+  const authScripts = useMemo(() => {
+    const filteredScripts = Array.isArray(scripts)
+      ? scripts
+          .filter((item) => item && item.scriptType === 'person_authentication' && item.enabled)
+          .map((item) => item.name)
+          .filter(Boolean)
+      : []
+
+    // const agamaFlows = Array.isArray(agamaList)
+    //   ? agamaList.map((flow) => flow?.details?.projectMetadata?.projectName).filter(Boolean)
+    //   : []
+    const agamaFlows = []
+
+    const result = [...filteredScripts, SIMPLE_PASSWORD_AUTH, ...agamaFlows]
+
+    console.log('Auth scripts with Agama flows:', result)
+    return result
+  }, [scripts, agamaList])
```

Comment:
De-dupe/sort options and drop debug log.

Avoid duplicates (e.g., built-in vs script with same name) and remove console noise.

```diff
-    const result = [...filteredScripts, SIMPLE_PASSWORD_AUTH, ...agamaFlows]
-
-    console.log('Auth scripts with Agama flows:', result)
-    return result
+    const result = Array.from(
+      new Set([...filteredScripts, SIMPLE_PASSWORD_AUTH, ...agamaFlows])
+    ).sort()
+    return result
```